### ISSUE TITLE
support for SpringBoot version 3 and up

### ISSUE THREAD

**Issue by bane-avenso** [sixhours-team/memcached-spring-boot#147]
Hello team,

I'm trying to migrate our project being at SpringBoot v2.7.5 to the latest **v3.0.2**, and using: 

```
<!-- Memcached for Spring Boot -->
<dependency>
    <groupId>io.sixhours</groupId>
    <artifactId>memcached-spring-boot-starter</artifactId>
    <version>2.4.3</version>
    <exclusions>
        <exclusion>
            <groupId>com.googlecode.xmemcached</groupId>
            <artifactId>xmemcached</artifactId>
        </exclusion>
    </exclusions>
</dependency>
<dependency>
    <groupId>com.amazonaws</groupId>
    <artifactId>elasticache-java-cluster-client</artifactId>
    <version>1.2.0</version>
</dependency>
```

...but integration is not working.

Do you plan to provide an version upgrade soon maybe?

Thank you very much :-)

**Comment by igorbolic** [comment]
We are planing to add the required setup to support **Spring Boot 3**, sometime during next week.

In the meantime until it is released, you could still continue to use auto-configuration with **Spring Boot 3** using the following setup:

```
@Configuration(proxyBeanMethods = false)
@ImportAutoConfiguration({
        io.sixhours.memcached.cache.MemcachedCacheAutoConfiguration.class
})
public class SomeConfiguration {
...
}
```

**Comment by igorbolic** [comment]
Fixed with release version **2.4.4**
